documents/documents.html

```diff
@@ -137,8 +138,66 @@ <h2>Installation Guide</h2>
             </iframe>
           </div>
         </section>
-        <section id="typography">
-         
+        <section id="typography" class="component-container">
+          <h2>Typography</h2>
+          <p class="mt-1 fs-2 grey-text">
+            dreamUI provides text-utlity classes which you can directly use in your project
+          </p>
+          <p class="fs-2 mt-2">Headings:</p>
+          <p class="pt-1">
+           All HTML headings from 
+            <span class="orange-theme-color">h1 to h6</span> are available
+          </p>
+          <div class="typography-container ">
+            <h1>h1 heading dreamUI</h1>
+            <h2>h2 heading dreamUI</h2>
+            <h3>h3 heading dreamUI</h3>
+            <h4>h4 heading dreamUI</h4>
+            <h5>h5 heading dreamUI</h5>
+            <h6>h6 heading dreamUI</h6>
+          </div>
+          <div class="code-snippet">
+            <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%2821%2C23%2C24%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch1%253Eh1%2520heading%2520dreamUI%253C%252Fh1%253E%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch2%253Eh2%2520heading%2520dreamUI%253C%252Fh2%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%253Eh3%2520heading%2520dreamUI%253C%252Fh3%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch4%253Eh4%2520heading%2520dreamUI%253C%252Fh4%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch5%253Eh5%2520heading%2520dreamUI%253C%252Fh5%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch6%253Eh6%2520heading%2520dreamUI%253C%252Fh6%253E"
+  sandbox="allow-scripts allow-same-origin">
```

Comment:
A code-formatter can be used to beautify the code!

Reply:
Formatted